packages/mermaid/src/diagrams/git/gitGraphRenderer.ts

```diff
@@ -899,9 +901,12 @@ export const draw: DrawDefinition = function (txt, id, ver, diagObj) {
   clear();
 
   log.debug('in gitgraph renderer', txt + '\n', 'id:', id, ver);
+  const DEFAULT_CONFIG = getConfig();
+  const DEFAULT_GITGRAPH_CONFIG = DEFAULT_CONFIG?.gitGraph;
   if (!DEFAULT_GITGRAPH_CONFIG) {
     throw new Error('GitGraph config not found');
   }
```

Comment:
As this is not the default config, using this name is misleading. You can use `config` or `gitGraphConfig`.